concordium-contracts-common/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+- Add `Display` trait to `SchemaType`
```

Comment:
I think it is important to emphasize that this is displaying the schema type as a JSON template